tests/llmobs/test_llmobs_span_agentless_writer.py

```diff
@@ -31,6 +32,16 @@ def test_buffer_limit(mock_writer_logs, mock_http_writer_send_payload_response):
         )
 
 
+def test_flush_queue_when_event_cause_queue_to_exceed_payload_limit(
+    mock_writer_logs, mock_http_writer_send_payload_response
+):
+    with override_global_config(dict(_dd_api_key="foobar.baz", _dd_site=DATADOG_SITE)):
+        llmobs_span_writer = LLMObsSpanWriter(is_agentless=True, interval=1000, timeout=1)
+        llmobs_span_writer.enqueue(_large_event())
+        llmobs_span_writer.enqueue(_large_event())
+        mock_writer_logs.debug.assert_has_calls([mock.call("encode %d LLMObs span events to be sent", 1)])
```

Comment:
Can we add an assertion that the debug log `"flushing queue because queuing next event will exceed EVP payload limit"` was called?